docs/modules/vertiq_23xx_family.rst

```diff
@@ -0,0 +1,145 @@
+
+.. _vertiq_23xx_family:
+
+*********************************
+Vertiq 23-XX Family 
+*********************************
+
+.. image:: ../_static/module_pictures/23xx_family.png
+        :alt: Vertiq 23-XX Family
+        :width: 500
+        :align: center
+
+.. csv-table:: Vertiq 23-XX Family of Modules
+        :header: "Size", "Kv", "Default Firmware", "Available Firmware"
+        :align: center
+
+        "23-06", "220", "Servo", "Servo"
+        "23-06", "2200", "Speed", "Speed, Pulsing"
+        "23-14", "920", "Speed", "Servo, Speed"
+        
+Hardware Setup Walkthrough
+===============================
+
+Changing or Updating Firmware
+===============================
+
+Please follow the instructions found in :ref:`updating_firmware`
+
+Getting Started
+===============================
+
+Complete the correct "Getting Started Guide" for your module's style (if you are using a servo module, please complete "Getting Started with Vertiq Servo Modules")
```

Comment:
Is the "(if you are using a servo module, please complete "Getting Started with Vertiq Servo Modules")" comment at the end here necessary? May be redundant